lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -1935,6 +1935,11 @@
         ObjectDefineProperty(Intl, "DateTimeFormat",        { value: DateTimeFormat,        writable: true, enumerable: false, configurable: true });
     }
 
+    // used for testing platform APIs, disabled by default
+    if (platform.showPlatform === true) {
+        ObjectDefineProperty(Intl, "platform", { value: platform });
```

Comment:
Wait... you're using JS to plumb the platform object through to Intl.platform? I guess it's the same either way but why not do this from the native code after Intl initialization (platform object should be the same as what was passed into Intl, right?).

I guess this lets you change platform from JS and have those changes reflected in the tests.

(FWIW I'm fine with this.)

Reply:
Yeah, I suppose that works too. Probably simpler. This was just the first way that I thought to do it.